contracts/staking/TokenStaking.sol

```diff
@@ -51,9 +47,9 @@ contract TokenStaking is Initializable, IStaking, Checkpoints {
     }
 
     struct StakingProviderInfo {
-        uint96 nuInTStake;
+        uint96 legacyNuInTStake;
         address owner;
-        uint96 keepInTStake;
+        uint96 legacyKeepInTStake;
```

Comment:
Can we leave the old names? I ran the OZ upgrade tool on the new contract (see below), and so far the only thing it doesn't like is this name change. For variable names, there's a way to override this error, but for the moment not for struct members ( see https://docs.openzeppelin.com/upgrades-plugins/1.x/faq#how-to-rename). For the moment, instead of changing the name we can leave a comment.

OZ output:
```
Error: ERROR processing /Users/david/dev/t/solidity-contracts/deploy/60_validate_upgrade_token_staking.ts:
Error: New storage layout is incompatible

contracts/staking/TokenStaking.sol:120: Upgraded `stakingProviders` to an incompatible type
  - In mapping(address => struct TokenStaking.StakingProviderInfo)
    - Bad upgrade to struct TokenStaking.StakingProviderInfo
  - In struct TokenStaking.StakingProviderInfo
    - Renamed `nuInTStake` to `legacyNuInTStake`
    - Renamed `keepInTStake` to `legacyKeepInTStake`
```

Side-note: I need to add this validation tool to CI, probably I'm doing it as part of #140. I also need to check why the tool doesn't complain about the change in the `Staked` event signature.

Reply:
Changing of event can't break storage but renaming theoretically can (if we would use those variables) somewhere. 
Renamed back ✔️